### ISSUE TITLE
Question about parameter setting on num_epoch

### ISSUE THREAD

**Issue by hzlihang99** [RinneSz/CLGA#6]
Hi,

Thanks for providing code for your great work. I have a question about the "num_epoch" parameter in CLGA.py. What is the ideal value for this parameter? I saw the sample command use 3,000, which cause it generate poisoned graph very slow, about every 4mins per edge on Cora. What value did you use for your paper's experiment?

Appreciate for your help.

**Comment by RinneSz** [comment]
It should not be that slow. Are you running it on GPU?

**Comment by hzlihang99** [comment]
Yes, I ran it with Nvidia-Tesla v100, and here are some screenshot for the running status. I found when is running, the program use both CPU (~23 cores) and GPU (~30% GPU utilities and 1.2 G GPU memory). It is that normal?

![Screen Shot 2023-09-18 at 11 19 44](https://github.com/RinneSz/CLGA/assets/16269383/129a8c9d-3ec9-4bfa-8f14-cfd03c0cf753)
![Screen Shot 2023-09-18 at 11 19 21](https://github.com/RinneSz/CLGA/assets/16269383/b65fdf7f-4fa4-45cd-a642-bcee8bc78e59)

![Screen Shot 2023-09-18 at 11 19 14](https://github.com/RinneSz/CLGA/assets/16269383/b356f180-cb5a-4d85-8306-f1ec0d1e2770)

**Comment by RinneSz** [comment]
What is the version of your python and pytorch?

**Comment by hzlihang99** [comment]
My python is 3.7.16 version. 

And the version of torch related package is listed as below: 
torch                         1.12.1+cu116
torch-cluster                 1.6.0+pt112cu116
torch-geometric               2.3.1
torch-scatter                 2.0.9
torch-sparse                  0.6.15+pt112cu116
torch-spline-conv             1.2.1+pt112cu116
torchaudio                    0.12.1+cu116
torchmetrics                  0.10.3
torchvision                   0.13.1+cu116

**Comment by hzlihang99** [comment]
Apart from that, I notice that the "num_epoch" parameter is used in the [`inner_train` function](https://github.com/RinneSz/CLGA/blob/434808c92fb6a8ab815733447f37a2fa2f6bedef/CLGA.py#L104C43-L104C53). I think if you use 3,000 for that, it is highly possible to have each edge cost 3min to finish, as running GCN for 3,000 times does cost so much time.
